drivers/hud_twenty_twenty_to_twenty_twenty_two/app/models/hud_twenty_twenty_to_twenty_twenty_two/client/db.rb

```diff
@@ -0,0 +1,18 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudTwentyTwentyToTwentyTwentyTwo::Client::Db
+  module_function
+
+  def up
+    HudTwentyTwentyToTwentyTwentyTwo::Client::Transform.up(
+      HudTwentyTwentyToTwentyTwentyTwo::Kiba::RailsSource,
```

Comment:
So rails was separate from `postgres` even though rails is backed by PG?

Reply:
I decided to implement using ActiveRecord, so I guess, technically, these are AR/PG (the destination uses AR-import with `on_duplicate_key_update` to do batch updates).